mne/channels/montage.py

```diff
@@ -14,6 +14,7 @@
 import os.path as op
 
 import numpy as np
+import xml.etree.ElementTree as et
```

Comment:
let's leave this as `ElementTree` to be explicit. We don't use this anywhere else so when it appears in code `et` will be mysterious